subsys/bluetooth/Kconfig

```diff
@@ -196,6 +196,13 @@ config BT_CHANNEL_SOUNDING
 	help
 	  Enable support for Bluetooth 6.0 Channel Sounding feature.
 
+config BT_CHANNEL_SOUNDING_TEST
+	bool "Channel Sounding Test support"
```

Comment:
`select EXPERIMENTAL` should be accompanied by " [EXPERIMENTAL]" at the end of the option name.